recipes/cuda-api-wrappers/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "cuda-api-wrappers" %}
+{% set version = "0.7.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/eyalroz/cuda-api-wrappers/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: fa30c9fe43a62f5a3fd82a5deb477838fbf0bf455c73a2d2bb5ab6284184900b
+
+build:
+  number: 0
+  skip: True  # [cuda_compiler != 'None']
```

The failures are actually correct ;)
```suggestion
  skip: True  # [cuda_compiler == 'None']
```